src/Uri.php

```diff
@@ -182,6 +182,30 @@ public function addPath(null|string $path = null): Uri
     }
 
     /**
+     * Appends given path to current path value. Auto adds a slash if it needs to be added.
+     *
+     * @param  string $path
+     * @return self
+     */
+    public function 
```

Comment:
Hey @JustSteveKing thanks for looking to it, i see that something went wrong here.

Also I don't see the tests I've thought I've added 🤔